Pages/Page 2_Big Picture.md

```diff
@@ -0,0 +1,170 @@
+        7  pos-2-5-0-0-cr01.dallas.tx.ibone.comcast.net (68.86.85.25)  64.643 ms  64.659 ms  64.653 ms
+        8  he-3-14-0-0-cr01.dallas.tx.ibone.comcast.net (68.86.85.1)  67.039 ms  67.079 ms  67.056 ms
+        9  he-0-10-0-0-pe07.ashburn.va.ibone.comcast.net (68.86.83.66)  63.013 ms  67.045 ms  88.273 ms
+        10  * * *
+        11  * * *
+        12  * * *
+        13  * * *
+        14  * * *
+        15  * * *
+        16  * * *
+        17  * * *
+        18  * * *
+        19  * * *
+        20  * * *
+        21  * * *
+        22  * * *
+        23  * * *
+        24  * * *
+        25  * * *
+        26  * * *
+        27  * * *
+        28  * * *
+        29  * * *
+        30  * * *
+
++ Line 1 is my computer server address.
++ Line 2 is my router server address.
++ Lines 3-30, the request is moving through Comcast's network of servers to get the information.
+
+When you enter [https://github.com](httpss://github.com) in the address bar, your browser makes a GET request to the GitHub server. There are several types of HTTP requests, but GET is the one that asks the server to send data from a specified resource.  The server sends back an HTML page and the Web browser turns the HTML into the web page you see through a process called rendering. You can see the HTML by right-clicking on the page and selecting `View Page Source`.
+
+
+### HTML Proper
+
+
+HTML stands for "HyperText Markup Language".
+
+Hypertext means documents can contain links to other pages or images. The structure of the HTML document is encoded using a markup language consisting of opening and closing elements (also called tags). Each segment of text is formatted according to the type of tag (`<body>, <title>, <h2>,` etc). Each segment of text needs an opening tag, `<body>`, and a closing tag, `</body>`. The `/` signifies the end of the formatting for that segment.
+
+#### Let's see it in action
+Open a text editing program and enter the following text: 
+
+`<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample HTML Page</title>
+  </head>
+  <body>
+    This is a sample html page.  This is more text.
+  </body>
+</html>`
+
+Save the file as 'sample.html'. Then open the file in the web browser. On Windows or a Mac, double-click the file to open it in the default browser, or right-click and select `Open With`. In your browser you should see something like this,
+
+* <<<_insert screenshot_>>>
+
+Notice the address bar. Instead of making an HTTP request to a server over the Internet, the browser  opened a local file and displayed the HTML. Remember, the web browser renders the HTML to make it appear as you see it in the browser. Right-click the page and select `View Page Source` to see the HTML elements of the file you saved.
+
+The first line of the source, the DOCTYPE, announces the document is HTML.  The text is enclosed between the _opening_ `<html>` and _closing_ `</html>` tags, or elements.
+
+#####Titles, Headers, & Body Text
```

Comment:
Does it affect the rendering to not have a space between the hashes and the title?